app/javascript/controllers/extraction_controller.js

```diff
@@ -26,12 +26,20 @@ export default class extends Controller {
     Turbo.navigator.delegate.adapter.progressBar.hide()
     document.querySelector('html').setAttribute('data-ai-loading', false)
 
+    this.pasteData(response)
+  }
+
+  pasteData(response) {
```

Comment:
by "pasting" do you mean filling the form?

Reply:
Yes